build/charts/antrea/conf/antrea-controller.conf

```diff
@@ -74,6 +74,11 @@ tlsCipherSuites: {{ .Values.tlsCipherSuites | quote }}
 # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
 tlsMinVersion: {{ .Values.tlsMinVersion | quote }}
 
+apiserverAuthentication:
+{{- with .Values.apiserverAuthentication }}
+  clientCAFile: {{ .clientCAFile | quote }}
+{{- end }}
```

Comment:
I feel apiserverAuthentication is a too narrowed scope which can hardly group other configurations. It could just be apiserver or in the root group directly given apiserver is a generic component and not specific to one feature, and the other similar configurations tlsCipherSuites, tlsMinVersion are already in the root group.

Reply:
updated.